chef/automate
# Event Feed Download

### :nut_and_bolt: Description: What code changed, and why?

* Adds a new endpoint that allows CSV and JSON download of the events.
* Moves the event feed protobufs from the automate gateway to the api/external directory
* Adds the chef org and chef server URL to the event feed API. 

### :chains: Related Resources

#4013
#4158 
#4016 

### :athletic_shoe: How to Build and Test the Change
1. `build components/event-feed-service && build components/automate-gateway && start_all_services`
1. Add some chef server events that have the chef org and chef server URL set. Run this command a few times `event_feed_add_event`. 
1. To download the event as CSV run `wget -O event_feed.csv --no-check-certificate --post-data='{"output_type": "csv"}' --header "api-token: $(get_api_token)" https://localhost/api/v0/eventfeed/export`
1. To download the event as JSON run `wget -O event_feed.json --no-check-certificate --post-data='{"output_type": "json"}' --header "api-token: $(get_api_token)" https://localhost/api/v0/eventfeed/export`
1. Look at the event_feed.csv and event_feed_json and ensure they have the same data as the https://a2-dev.test/dashboards/event-feed page. 
1. Run this curl command to ensure the chef org and chef server URL is returned in the event feed API. `curl -sf --insecure -H "api-token: $(get_api_token)" "https://localhost/api/v0/eventfeed?collapse=true&page_size=100" | jq` Ensure the "chef_organization" and "chef_infra_server" fields exists for each event. 

### :white_check_mark: Checklist


### :camera: Screenshots, if applicable
